src/NewUser/NewUser.tsx

```diff
@@ -59,7 +59,7 @@ const NewUser: React.FC = () => {
             password: password,
             password_confirmation: confirmPassword
         }
-        if(evaluateEmail(email) && evaluatePassword(password, confirmPassword) && evaluateUsername(userName)){
+        if(evaluateEmail(email)  && evaluateUsername(userName) && evaluatePassword(password, confirmPassword)){
```

Comment:
`validate` may be a better prefix for these functions.

Reply:
Valid point